src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="fieldset-block">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text"
+                   name="lastname"
+                   autocomplete="off"
+                   class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text"
+                   name="firstname"
+                   autocomplete="off"
+                   class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input type="number"
+                   name="age"
+                   value="12"
+                   min="1"
+                   max="100"
+                   class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date"
+                 name="bday"
+                 class="input-margin"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox"
+                 name="agree"
+                 class="input-margin"
+          >
+        </label>
+
+      </fieldset>
```

Comment:
```suggestion
        </label>
      </fieldset>
```

No need to add empty line between parent and child elements